packages/dara-core/dara/core/interactivity/any_variable.py

```diff
@@ -134,13 +134,19 @@ def restore_contexts():
             )
             return None
 
-        session_channels: Dict[str, str] = {}
+        session_channels: Dict[str, Set[str]] = {}
+        saved_ws_channel = WS_CHANNEL.get()
 
         # Collect sessions which are active
         for sid in session_ids:
             if websocket_registry.has(sid):
                 ws_channel = websocket_registry.get(sid)
```

Comment:
Can we rename this to `ws_channels`, it's a set so gets a little confusing to read